Console/Command/QueueShell.php

```diff
@@ -11,7 +11,9 @@
 
 App::uses('Folder', 'Utility');
 App::uses('File', 'Utility');
-App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
+if (!class_exists('HipChat\HipChat')) {
```

Comment:
App::uses didn't work as expected in this case.  I think it has to do with the namespaces.  It was still trying to include the file twice.